virt/virtual_machines/advanced_vm_management/virt-automatic-bootsource-updates.adoc

```diff
@@ -12,6 +12,10 @@ include::modules/virt-enabling-bootsource-update.adoc[leveloffset=+1]
 
 include::modules/virt-disable-bootsource-update.adoc[leveloffset=+1]
 
+include::modules/virt-disable-individual-bootsource-update.adoc[leveloffset=+1]
+
 include::modules/virt-reenabling-bootsource-update.adoc[leveloffset=+1]
 
+include::modules/virt-configuring-storage-class-bootsource-update.adoc[leveloffset=+1]
+
 include::modules/virt-autoupdate-custom-bootsource.adoc[leveloffset=+1]
```

Comment:
A small question, not sure whether it is better to adjust the order for this, eg:
include::modules/virt-enabling-bootsource-update.adoc[leveloffset=+1]
include::modules/virt-disable-bootsource-update.adoc[leveloffset=+1]
include::modules/virt-reenabling-bootsource-update.adoc[leveloffset=+1]
include::modules/virt-configuring-storage-class-bootsource-update.adoc[leveloffset=+1]
include::modules/virt-autoupdate-custom-bootsource.adoc[leveloffset=+1]
include::modules/virt-disable-individual-bootsource-update.adoc[leveloffset=+1]

Reply:
Hi @duyanyan - I made these changes, as you probably know the workflow better than I do. My only question is if `disable individual `should be after `disable bootsource` as you are doing the same thing, first for all, then for one. If you don't agree, that's fine, just asking. Thanks, Bob

Reply:
Actually, I think your way is best the more I think about it. So, let's do that. I wasn't thinking in terms of predefined/custom but that makes a lot of sense.